exercises/lifetimes/lifetimes3.rs

```diff
@@ -5,11 +5,9 @@
 // Execute `rustlings hint lifetimes3` or use the `hint` watch subcommand for a
 // hint.
 
-// I AM NOT DONE
-
-struct Book {
-    author: &str,
-    title: &str,
+struct Book<'a> {
```

Comment:
Notice you are forcing both lifetimes to be the same for author and title.
Why?

Reply:
Since all variables in this struct must live for the same duration, we are printing them at the end of the main in the same place.